src/test/java/org/eclipse/emfcloud/jackson/tests/ArrayTest.java

```diff
@@ -0,0 +1,148 @@
+package org.eclipse.emfcloud.jackson.tests;
```

Comment:
Could you please add a matching copyright header at the top of this file?

```
/*******************************************************************************
 * Copyright (c) 2022 <your-company-or-name> and others.
 *
 * This program and the accompanying materials are made available under the
 * terms of the Eclipse Public License v. 2.0 which is available at
 * https://www.eclipse.org/legal/epl-2.0, or the MIT License which is
 * available at https://opensource.org/licenses/MIT.
 *
 * SPDX-License-Identifier: EPL-2.0 OR MIT
 *******************************************************************************/
```

Reply:
Done